src/Apps/Sell/Routes/__tests__/ThankYouRoute.jest.tsx

```diff
@@ -0,0 +1,15 @@
+import { ThankYouRoute } from "Apps/Sell/Routes/ThankYouRoute"
+import { mount } from "enzyme"
+
+describe("ThankYouRoute", () => {
+  it("renders text and links", () => {
+    const wrapper = mount(<ThankYouRoute />)
```

Comment:
To make sure the component renders without any issues when wrapped with `SubmissionRoute`.

```suggestion
const { renderWithRelay } = setupTestWrapperTL({
  Component: (props: any) => {
    return (
      <SubmissionRoute submission={props.submission}>
        <ThankYouRoute />
      </SubmissionRoute>
    )
  },
  query: graphql`
    query ArtistRoute_Test_Query @raw_response_type {
      submission(id: "submission-id") {
        ...SubmissionRoute_submission
      }
    }
  `,
})
```